manager/Makefile

```diff
@@ -26,14 +26,14 @@ docker-build: generate
 
 # Deploy only movement-controller in the configured Kubernetes cluster in ~/.kube/config
 deploy_mc: manifests $(TOOLBIN)/kustomize $(TOOLBIN)/kubectl
-	$(TOOLBIN)/kubectl create namespace m4d-system || true
+	$(TOOLBIN)/kubectl create namespace ${DOCKER_NAMESPACE} || true
```

Comment:
Tying this to the DOCKER_NAMESPACE is limiting and confusing. Just because it works without extra work in the internal openshift registry doesn't mean that we should force it to be like that. Instead, **if** you use the OpenShift internal registry then set DOCKER_NAMESPACE to the same value as `KUBE_NAMESPACE` (or an equivalent).